k8s/crds/kops.k8s.io_clusters.yaml

```
@@ -2407,6 +2407,31 @@ spec:
                   description: AmazonVPCNetworkingSpec declares that we want Amazon
                     VPC CNI networking
                   properties:
+                    env:
+                      description: Env is a list of environment variables to set in
+                        the container.
+                      items:
+                        description: EnvVar represents an environment variable present
+                          in a Container.
+                        properties:
+                          name:
+                            description: Name of the environment variable. Must be
+                              a C_IDENTIFIER.
+                            type: string
+                          value:
+                            description: 'Variable references $(VAR_NAME) are expanded
```

This is a bit awkward, imho. Perhaps something along these lines? (If i've understood correctly)
```suggestion
                            description: Variables are expanded by merging the default container variables and your custom variables. When a reference cannot be resolved, the input will be unchanged and will contain `$(VARIABLES)`. If a variable should never be expanded, use the format `$$(VARIABLE)` which will render as `$(VARIABLE)`.
```